### ISSUE TITLE
fix encoding problems for comments and summaries

### ISSUE THREAD

**Issue by mindbergh** [austinlostinboston/mitsWebApp#19]
A lot of comments and summaries are displayed with back slashes and other unescaped punctuations. We don't want that, do we?

**Comment by WangWenjun559** [comment]
yes, we don't want that. Maybe Austin can do something in response generator
